src/hooks/useEffectAfterMount.ts

```diff
@@ -0,0 +1,14 @@
+import { EffectCallback, useEffect, useRef } from "react";
```

Comment:
can you change the import a bit to `import { type EffectCallback, useEffect, useRef }`